redis-pq-demo/README.md

```diff
@@ -0,0 +1,29 @@
+# Example run-time
+
+This is a demo that builds and deploys an instance of acapy that is configured to act as a mediator, and use REDIS to as a resilient method of managing acapy's event queue. 
+
+How to see a full fledged demo, running a local ledger, and a set of agents using the mediator (with redis)  
+
+To run a redis-cluster and acapy as a mediator with redis. 
+```
+git clone https://github.com/hyperledger/aries-mediator-service.git z
+cd redis-pq-demo
+docker-compose -f docker-compose.redis.yml up
```

Comment:
should mention that you need multiple terminals. should also mention how to stop and remove these instances.

Reply:
Added `-d` to the script so you don't need an extra terminal. good catch.